README.md

```diff
@@ -356,5 +356,114 @@ function `bodn2c`, and wants the vector version if it exists (but it doesn't!),
 they can call
 
 ```python
-cspyce.bodn2c.error.vector(args...)
+cspyce.bodn2c.error.vector(args, ...)
 ```
+
+## RECORD ENHANCEMENTS
```

Comment:
```suggestion
---
## RECORD ENHANCEMENTS
```

Reply:
Done

Reply:
I don't understand this fix. You didn't make the change I asked for but seem to have put in a different line in the wrong place.